src/plugins/intel_cpu/src/config.h

```diff
@@ -48,6 +48,7 @@ struct Config {
     size_t rtCacheCapacity = 5000ul;
     InferenceEngine::IStreamsExecutor::Config streamExecutorConfig;
     InferenceEngine::PerfHintsConfig  perfHintsConfig;
+    std::string core_type_cfg = "ANY_CORE";
```

Comment:
why not `ov::hint::scheduling_core_type ` enum?

Reply:
updated